package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "express-generator",
   "description": "Express' application generator",
-  "version": "4.13.1",
+  "version": "4.14.0",
```

Comment:
Ok. We should add this fact to the contribution guidelines doc.

/cc @tunniclm